class CustomException(Exception):
    def __init__(self, x, y):
        super().__init__()
